web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -71,16 +82,62 @@ Proposing a PDEP is done by creating a PR adding a new file to `web/pdeps/`.
 The file is a markdown file, you can use `web/pdeps/0001.md` as a reference
 for the expected format.
 
-The initial status of a PDEP will be `Status: Under discussion`. This will be changed to
-`Status: Accepted` when the PDEP is ready and has the approval of the core team.
+The initial status of a PDEP will be `Status: Draft`. This will be changed to
+`Status: Under discussion` by the author(s), when they are ready to proceed with the decision
+making process.
 
-#### Accepted PDEP
+#### PDEP Discussion Timeline
+
+A PDEP discussion will remain open for up to 60 days. This period aims to enable participation
+from volunteers, who might not always be available to respond quickly, as well as provide ample
+time to make changes based on suggestions and considerations offered by the participants.
+Similarly, the following voting period will remain open for 15 days.
+
+To enable and encourage discussions on PDEPs, we follow a notification schedule. At each of the
+following steps, the pandas team, and the pandas dev mailing list are notified via GitHub and
+E-mail:
+- Once a PDEP is ready for discussion.
+- After 30 discussion days, with 30 days remaining for discussion.
+- After 45 discussion days, with 15 days remaining for discussion.
+- In case 15 days passed without any new comments, the authors may close the discussion period
+  and open the voting period.
+- Once the voting period starts, after 60 days or in case of an earlier vote, with 15 days
+  remaining for voting.
+- After 10 voting days, with 5 days remaining for voting.
+
+#### Casting Votes
+
+As the voting period starts, a VOTE issue is created which links to the PDEP discussion pull request.
+Each voting member may cast a vote by adding one of the following comments:
 
-A PDEP can only be accepted by the core development team, if the proposal is considered
-worth implementing. Decisions will be made based on the process detailed in the
-[pandas governance document](https://github.com/pandas-dev/pandas-governance/blob/master/governance.md).
-In general, more than one approval will be needed before the PR is merged. And
-there should not be any `Request changes` review at the time of merging.
+- +1: approve.
+- 0: abstain.
+  - Reason: A one sentence reason is required.
+- -1: disapprove
+  - Reason: A one sentence reason is required.
+A disapprove vote requires prior participation in the PDEP discussion issue.
+
+Once the voting period ends, any voter may tally the votes in a comment, using the format: x-y-z,
+where x stands for the total of approving, y of abstaining, and z of disapproving votes cast.
+
+#### Quorum and Majority
+
+For a PDEP vote to result in accepting the proposal, a quorum is required. All votes (including
+abstentions) are counted towards the quorum. The quorum is computed as the lower of these two
+values:
+
+- 11 voting members.
+- 50% of voting members.
+
+Given a quorum, a majority of 75% of the non-abstaining votes is required as well, i.e. 75% of
```

Comment:
Scikit learn seems to require a simple majority

Reply:
We discussed this in our governance meeting today, and at least within the group today, we all found the 75% appropriate. 

The 75% came from a consensus seeking decision-making approach, see [this wiki](https://en.wikipedia.org/wiki/Consensus_decision-making#Decision_rules).
The idea was that we don't to have a decision which had many negative votes. 

Personally, I also like the "unanimity minus three" approach - so that once 4 people are against something, the group shouldn't accept it. But I think it needs to appeal to everyone, so it would be great to get more opinions.

Reply:
We also though it could be helpful to make the 75% number more tangible with a concrete example: if 12 people vote, 4 negative votes would block it; if 16 people vote, 5 negative votes would block it.

(and so this current proposal is less strict than a "unanimity minus three" rule)